cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/sqlite/SQLQueueCallable.java

```diff
@@ -22,54 +22,59 @@
  *
  * @api_private
  */
-public abstract class SQLQueueCallable<T> implements Callable<T> {
+public class SQLQueueCallable<T> implements Callable<T> {
```

Comment:
Since this wrapper is only used by the `SQLDatabaseQueue` now it can be made less visible.